src/components/Welcomes/QRScanner/ShapeDetectionPolyfill.ts

```diff
@@ -1,9 +1,17 @@
-import jsQR from 'jsqr'
-import { isNull } from 'lodash-es'
 /// <reference path="./ShapeDetectionSpec.d.ts" />
+import { isNull } from 'lodash-es'
+
+const noop = () => {}
 
 class BarcodeDetectorPolyfill implements BarcodeDetector {
-    constructor() {}
+    private worker: Worker = new Worker(browser.runtime.getURL('js/qrcode.js'))
```

Comment:
We have a `getUrl` in the project that works for both WebExt and storybook. Use browser.runtime.getURL directly will cause this code to fail in the storybook